src/t8_schemes/t8_default/t8_default_common/t8_default_common.hxx

```diff
@@ -74,60 +221,37 @@ class t8_default_scheme_common_c: public t8_eclass_scheme_c {
    * Note that this number is >= 1, since we count the element itself as a sibling.
    */
   int
-  t8_element_num_siblings (const t8_element_t *elem) const override;
+  element_get_num_siblings (const t8_element_t *elem) const
+  {
+    const int dim = t8_eclass_to_dimension[eclass];
+    T8_ASSERT (eclass != T8_ECLASS_PYRAMID);
+    return sc_intpow (2, dim);
+  }
 
   /** Count how many leaf descendants of a given uniform level the root element will produce.
    * \param [in] level A refinement level.
    * \return The value of \ref t8_element_count_leaves if the input element
    *      is the root (level 0) element.
    */
   t8_gloidx_t
```

Comment:
Please add a comment that pyramids have their own implementation of this function.